### PR TITLE
Vmo 4403 fix toolbar position with scrolling

### PR DESCRIPTION
**Pull request by safydy-r**
**Note for devs:**
- To make sure the toolbar keeps the position `sticky` and behaves like `fixed`, I used this solution: https://stackoverflow.com/a/57452861/3256489
- I took the opportunity to silent some `NavigationDuplicated` errors like we do with validations https://github.com/FLOIP/flow-builder/blob/VMO-4403-fix-flow-builder-block-editor-position/src/components/interaction-designer/toolbar/ErrorNotifications.vue#L133

**Screenshot**
vmo-4403

### CODE REVIEW

File: src/components/interaction-designer/BuilderCanvas.vue [hunk 1]
```diff
@@ -186,7 +187,7 @@ export default class BuilderCanvas extends Vue {
       return this.windowWidth - this.widthAdjustment
     }
 
-    const xPosition: number = this.blockAtTheLowestPosition.ui_metadata.canvas_coordinates.x
+    const xPosition: number = this.blockAtTheFurthestRightPosition.ui_metadata.canvas_coordinates.x
```
Comment (safydy-r) [inline src/components/interaction-designer/BuilderCanvas.vue:190]: fixed wrong copy-paste from `canvasHeight()`

Comment (poojahpatil90) [reply src/components/interaction-designer/BuilderCanvas.vue:190]: Can we use this opportunity to start the first block from the left-top corner or center. Right now it starts from the bottom-right corner. Suggesting this as it feels like this is the right spot to make the change.
If it is a time consuming change, then we can handle it later.

**Review by poojahpatil90 — COMMENTED** [review]
@safydy-r  I tested this fix by checking out your branch. For some reason it does not seem to be working on my machine. Here is a screen capture to demonstrate the same.  Let me know if you feel I might be missing something.

File: src/components/interaction-designer/BuilderCanvas.vue (hunk 1 above)
Comment (safydy-r) [reply src/components/interaction-designer/BuilderCanvas.vue:190]: @poojahpatil90 , I'd like to handle that later in a different ticket.

**Review by poojahpatil90 — APPROVED** [review]
Thanks for fixing the issue. Looks good to me now. Approved!

**Pull request merged**